tests/main/security-device-cgroups-helper/task.yaml

```diff
@@ -4,8 +4,9 @@ details: |
   The test verifies that snap-device-helper correctly modifies the cgroups
 
 systems:
-  #TODO: bpftool is not available on core22 and tests.device-cgroup needs it for cgroups v2
+  #TODO: bpftool is not available on core22+ and tests.device-cgroup needs it for cgroups v2
   - -ubuntu-core-22-*
+  - -ubuntu-core-24-*
```

Comment:
Not sure if we keep track of this in some way but maybe we could mark these places that need to be updated with an `#UPDATE` comment. When adding a new system, we could grep for that and find what needs updating